cloudify_cli/commands/snapshots.py

```diff
@@ -25,64 +25,57 @@
 def restore(snapshot_id, without_deployments_envs, force):
     logger = get_logger()
     management_ip = utils.get_management_server_ip()
-    logger.info("Restoring snapshot '{0}' at management server {1}"
-                .format(snapshot_id, management_ip))
+    logger.info('Restoring snapshot {0}...'.format(snapshot_id))
     client = utils.get_rest_client(management_ip)
     execution = client.snapshots.restore(
         snapshot_id, not without_deployments_envs, force)
-    logger.info('Started workflow\'s execution id: {0}'.format(execution.id))
+    logger.info('Started workflow execution (id: {0})'.format(execution.id))
```

Comment:
No where above have you put "x process (id: {0})" in the message it's usually like "deployment {0}" so it should be "execution id: {0}"